### ISSUE TITLE
Second parameter ($attributes) missing

### ISSUE THREAD

**Issue by Donkfather** [makeabledk/laravel-factory-enhanced#2]
Hi. 
In FactoryBuilder:376 -> mergeAndExpandAttributes in call_user_func there should be a second parameter with the $attributes. I had my defineAs accepting the attributes as the second parameter in the callback and after i installed this package it blew up.
I will come back with more details.

Laravel version: 5.7
Package version: v1.1

**Comment by rasmuscnielsen** [comment]
Thanks for submitting this issue! 

We'll have a look at this and submit a patch asap. 

Please do get back to us with more details whenever you find time, in case we haven't resolved this by then. 

Cheers

**Comment by rasmuscnielsen** [comment]
@Donkfather I've looked a bit further into this issue.

Is it correct that the missing second argument `attributes` should only contain the *additional* attributes passed to the ie. `create()` method?

```php
factory(Company::class)->create(['vat_no` => 12345678]);
```

```php
$this->factory()->define(Company::class, function (Generator $faker, array $attributes) {
    // $attributes only contains 'vat_no'
    return [
        'name' => $faker->company
    ];
});
```

**Comment by rasmuscnielsen** [comment]
@Donkfather 

I believe I've fixed this issue in the latest commit on `master` branch.

Before I tag a release, It'd be great if you could pull down `dev-master` and check if this has fixed your tests / seeders. 

Please let me know if this is possible for you, and if so, whether or not it works as expected :-)

**Comment by rasmuscnielsen** [comment]
Should be fixed in v1.2
